cartography/intel/gcp/databases.py

```diff
@@ -0,0 +1,864 @@
+    :param sql: The GCP Bigtable resource object
+
+    :type project_id: str
+    :param project_id: The project ID of the corresponding project
+
+    :type gcp_update_tag: timestamp
+    :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+
+    :type common_job_parameters: dict
+    :param common_job_parameters: Dictionary of other job parameters to pass to Neo4j
+
+    :rtype: NoneType
+    :return: Nothing
+    """
+    logger.info("Syncing GCP Cloud SQL for project %s.", project_id)
+    #BIGTABLE INSTANCES
+    bigtableinstances = get_bigtable_instances(bigtable,project_id)
+    load_bigtable_instances(neo4j_session,bigtableinstances,project_id,gcp_update_tag)
+    #BIGTABLE CLUSTERS
+    bigtableclusters = get_bigtable_clusters(bigtable,project_id)
+    load_bigtable_clusters(neo4j_session,bigtableclusters,project_id,gcp_update_tag)
+    #BIGTABLE CLUSTER BACKUPS
+    clusterbackups = get_bigtable_cluster_backups(bigtable,project_id)
+    load_bigtable_cluster_backups(neo4j_session,clusterbackups,project_id,gcp_update_tag)
+    #BIGTABLE TABLES
+    bigtabletables = get_get_bigtable_tables(bigtable,project_id)
+    load_bigtable_tables(neo4j_session,bigtabletables,project_id,gcp_update_tag)
+    # TODO scope the cleanup to the current project - https://github.com/lyft/cartography/issues/381
+    cleanup_bigtable(neo4j_session, common_job_parameters)
+    
+@timeit
+def sync_firestore(
+    neo4j_session: neo4j.Session, firestore:Resource, project_id: str, gcp_update_tag: int,
+    common_job_parameters: Dict
+) -> None:
+    """
+    Get GCP Cloud SQL Instances and Users using the Cloud Function resource object, ingest to Neo4j, and clean up old data.
+
+    :type neo4j_session: The Neo4j session object
+    :param neo4j_session: The Neo4j session
+    
+    :type firestore: The GCP Firestore resource object created by googleapiclient.discovery.build()
+    :param firestore: The GCP Firestore resource object
+
+    :type project_id: str
+    :param project_id: The project ID of the corresponding project
+
+    :type gcp_update_tag: timestamp
+    :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+
+    :type common_job_parameters: dict
+    :param common_job_parameters: Dictionary of other job parameters to pass to Neo4j
+
+    :rtype: NoneType
+    :return: Nothing
+    """
+    logger.info("Syncing GCP Cloud SQL for project %s.", project_id)
+    #FIRESTORE DATABASES
+    firestoredatabases = get_firestore_databases(firestore,project_id)
+    load_firestore_databases(neo4j_session,firestoredatabases,project_id,gcp_update_tag)
+    #FIRESTORE INDEXES
+    firestoreindexes = get_firestore_indexes(firestore,project_id)
+    load_firestore_indexes(neo4j_session,firestoreindexes,project_id,gcp_update_tag)
+    # TODO scope the cleanup to the current project - https://github.com/lyft/cartography/issues/381
+    cleanup_firestore(neo4j_session, common_job_parameters)
```

Comment:
we should add a method like `sync_databases` which would orchestrate sync of sql, bigtable and firestore, etc. Otherwise, there's no single entry point to databases.py

Reply:
Ignore these. See the next comment